vuepress/docs/next/tutorials/solution/pda-architecture.md

```diff
@@ -5,7 +5,7 @@ deployment, as well as how they interact with each other.
 
 The image below shows the high level components.
 
-![PDA Architecture](./pda-images/pda-architecture.png)
+![PDA Architecture No Sidecar](./pda-images/pda-architecture-no-sidecar.png)
```

Comment:
Minor but let's remove -no-side-car from the file name. That's historical and this is simply the current pda-architecture.png. If we need the old one, we have it under the other versions.